sphinx/texinputs/latexmkjarc

```diff
@@ -0,0 +1,7 @@
+$latex = 'platex --halt-on-error --interaction=nonstopmode -kanji=utf8 %O %S';
+$dvipdf = 'dvipdfmx %O -o %D %S';
+$makeindex = 'rm -f %D; mendex -U -f -d %B.dic -s python.ist %S || echo "mendex exited with error code $? (ignoring)" && : >> %D';
```

Comment:
In the end, it was decided to keep usage to unix platforms, so the `rm -f %D` was re-added ...